swift/apple/FirezoneKit/Sources/FirezoneKit/Keychain/Keychain.swift

```diff
@@ -20,7 +20,7 @@ public enum KeychainError: Error {
 public actor Keychain {
   private let label = "Firezone token"
   private let description = "Firezone access token used to authenticate the client."
-  private let service = "dev.firezone.firezone"
+  private let service = Bundle.main.bundleIdentifier!
```

Comment:
This is updated just so that debug testing and release testing don't clobber each other's Keychain items.